pages/_app.tsx

```diff
@@ -1,24 +1,53 @@
-// Copyright 2020 the Deno authors. All rights reserved. MIT license.
+// Copyright 2022 the Deno authors. All rights reserved. MIT license.
 
-import React from "react";
-import App from "next/app";
-import Head from "next/head";
-import "../components/app.css";
-import "../components/markup.css";
-import "../components/docsearch.css";
-import "preact/debug";
+/** @jsx h */
+/** @jsxFrag Fragment */
+import { Fragment, h, Head } from "../deps.ts";
 
-export default class DenoWebsiteApp extends App {
-  render(): React.ReactElement {
-    const { Component, pageProps } = this.props;
-    return (
-      <div className="h-screen">
-        <Head>
-          <link rel="icon" href="/favicon.ico" sizes="32x32" />
-          <link rel="icon" href="/logo.svg" type="image/svg+xml" />
-        </Head>
-        <Component {...pageProps} />
-      </div>
-    );
-  }
+export default function App({ Component }) {
+  return (
+    <div class="h-screen">
+      <Head>
+        <link rel="icon" href="/favicon.ico" sizes="32x32" />
+        <link rel="icon" href="/logo.svg" type="image/svg+xml" />
+
+        <meta name="twitter:site" content="@deno_land" />
+        <meta name="twitter:creator" content="@deno_land" />
+        <meta
+          name="twitter:title"
+          content="Deno - A modern runtime for JavaScript and TypeScript"
+        />
+        <meta
+          name="twitter:description"
+          content="Deno is a simple, modern runtime for JavaScript and
+            TypeScript that uses V8 and is built in Rust."
+        />
+        <meta name="twitter:card" content="summary_large_image" />
+        <meta property="og:locale" content="en_US" />
+        <meta property="og:site_name" content="DenoLand" />
+        <meta
+          property="og:title"
+          content="Deno - A modern runtime for JavaScript and TypeScript"
+        />
+        <meta
+          property="og:description"
+          content="Deno is a simple, modern runtime for JavaScript and
+            TypeScript that uses V8 and is built in Rust."
+        />
+        <meta property="og:image" content="/images/icons/icon-512x512.png" />
+        <meta property="og:type" content="website" />
```

Comment:
Not for now but I'd really like to have custom `og:description` for each page in the future.  Current og behavior is pretty obnoxious and not very useful.